src/components/Home/Home.jsx

```diff
@@ -71,9 +95,13 @@ const Home = () => {
           <button type="submit">Search</button>
         </label>
       </form>
-      {listNotFound && <p>{listNotFound}</p>}
-      {error && <p>{error}</p>}
-      {load && <p>{load}</p>}
+      {/* {listNotFound && <p>{listNotFound}</p>}
+      {error && <p>{error}</p>} */}
+      {load && <p>{load.message}</p>}
+
+      {alertMessage && (
```

Comment:
Because `alertMessage` and `load` are both objects, they are both truthy unless we explicitly set them to `null`. So, we'll always be rendering these <p> tags in the DOM. Probably not an issue b/c they're empty strings, but just wanted to point that out.